web/src/components/Inputs/InputWrapper.js

```diff
@@ -0,0 +1,47 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+// a HOC for Text / Textarea input that includes
+// div wrapper and accompanying label
+const InputWrapper = InputInner => {
+  const InputHolder = ({
+    name,
+    label,
+    type,
+    hideLabel,
+    wrapperClass,
+    ...rest
+  }) => (
+    <div className={wrapperClass || 'mb2'}>
+      <label
+        htmlFor={name}
+        className={hideLabel ? 'sr-only' : 'block mb-tiny truncate'}
```

Comment:
I kinda think we should move toward defining classes as variables, like:
```
const labelClasses = {
  hidden: 'sr-only',
  visible: 'block mb-tiny truncate',
}

...

className={hideLabel ? labelClasses.hidden : labelClasses.visible}
```

That'd let us find and edit classes easier. But it's an optimization and an opinion, and def not something that needs to happen now!